src/components/app.scss

```diff
@@ -151,3 +151,7 @@
     max-width: 200px;
     text-align: left;
   }
+
+:global .react-spinners--clip {
```

Comment:
Can we not add a CSS style to the global scope?

(I guess this probably works if it's nested inside `.voyager` too.)